pyproject.toml

```
@@ -27,7 +27,7 @@ requires-python = ">= 3.8"
 # the same API.
 dependencies = [
     # Schema definition
-    "pydantic>=1.10.8,<2",
+    "pydantic>=2.5, <3",
```

Is there any reason to restrict the top requirement? More info why we we should be open by default: https://bernat.tech/posts/version-numbers/

```suggestion
    "pydantic>=2.5",
```